test/OpenTelemetry.Tests/Metrics/MetricViewTests.cs

```diff
@@ -78,6 +79,29 @@ public void AddViewWithInvalidNameThrowsArgumentException(string viewNewName)
             Assert.Contains($"Custom view name {viewNewName} is invalid.", ex.Message);
         }
 
+        [Theory]
+        [MemberData(nameof(MetricsTestData.ValidHistogramBounds), MemberType = typeof(MetricsTestData))]
+        public void AddViewWithValidHistogramBounds(int index, double[] bounds)
+        {
+            Sdk.CreateMeterProviderBuilder()
+                .AddView("name1", new HistogramConfiguration { BucketBounds = bounds })
+                .Build();
+        }
+
+        [Theory]
+        [MemberData(nameof(MetricsTestData.InvalidHistogramBounds), MemberType = typeof(MetricsTestData))]
+        public void AddViewWithInvalidHistogramBoundsThrowsArgumentException(int index, double[] bounds)
```

Comment:
This isn't throwing the exception currently. @cijothomas do you understand why not?